### PR TITLE
Migrate current and persistence footprint components

### PR DESCRIPTION
**Pull request by cazapatamar**

### CODE REVIEW

**Comment by erikasv** [comment]
Please connect the issue

File: src/utils/restAPI.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import axios, { CancelToken } from 'axios';
+import axios, { CancelToken } from "axios";
```
Comment (erikasv) [inline src/utils/restAPI.js:1]: Please undo the format changes in this file, it's hard to identify relevant changes.

You can configure code to format on save only ts and tsx:
```
  "[typescript]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.formatOnSave": true,
  },
  "[typescriptreact]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode",
    "editor.formatOnSave": true,
  },
```

File: src/utils/searchAPI.ts [hunk 2]
```diff
@@ -62,6 +67,57 @@ class SearchAPI {
...
+  /** HUMAN FOOTPRINT */
+  /** *************** */
+  /**
+  /**
+   * Get the current human footprint value in the given area.
+   *
+   * @param {String} areaType area type id, f.e. "ea", "states"
+   * @param {Number | String} areaId area id to request, f.e. "CRQ", 24
+   *
+   * @return {Object} Objecy with value and category for the current human footprint
```
Comment (erikasv) [inline src/utils/searchAPI.ts:80]: typo in `Objecy`

**Comment by cazapatamar** [comment]
> Please connect the issue

Done!

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/utils/restAPI.js (hunk 1 above)
Comment (cazapatamar) [reply src/utils/restAPI.js]: Fixed!

File: src/utils/searchAPI.ts (hunk 2 above)
Comment (cazapatamar) [reply src/utils/searchAPI.ts]: Fixed!

File: src/utils/restAPI.js [hunk 3]
```diff
@@ -175,87 +175,88 @@ class RestAPI {
...
    * @param {String} areaType area type id, f.e. "ea", "states"
    * @param {Number | String} areaId area id to request, f.e. "CRQ", 24
    *
-   * @return {Promise<Array>} Array of objects with data for the forest loss and persistence
+   * @return {Promise<Object>} Array of objects with data of the protected areas
    */
-  static requestForestLP(areaType, areaId) {
-    return RestAPI.makeGetRequest(`forest/lp?areaType=${areaType}&areaId=${areaId}`);
+  static requestDPC(areaType, areaId, paNumber) {
+    return RestAPI.makeGetRequest(`connectivity/dpc?areaType=${areaType}&areaId=${areaId}&paNumber=${paNumber}`);
```
Comment (erikasv) [inline src/utils/restAPI.js:259]: Why are these functions being restored, they were already migrated.

**Review by erikasv — CHANGES_REQUESTED** [review]

File: src/utils/restAPI.js (hunk 3 above)
Comment (cazapatamar) [reply src/utils/restAPI.js]: Fixed!

**Review by erikasv — APPROVED** [review]

**Pull request merged**